framework/src/problems/FEProblemBase.C

```diff
@@ -4245,6 +4245,44 @@ FEProblemBase::execMultiAppTransfers(ExecFlagType type, Transfer::DIRECTION dire
 
     _console << COLOR_CYAN << "\nStarting Transfers on " << Moose::stringify(type)
              << string_direction << "MultiApps" << COLOR_DEFAULT << std::endl;
+
+    // output transfer information
+    unsigned int max_name_length = 4;
+    unsigned int max_type_length = 4;
+    unsigned int max_multiapp_name_length = 13;
+    for (const auto & transfer : transfers)
+    {
+      if (transfer->name().length() > max_name_length)
+        max_name_length = transfer->name().length();
+      if (transfer->type().length() > max_type_length)
+        max_type_length = transfer->type().length();
+      auto multiapp_transfer = dynamic_cast<MultiAppTransfer *>(transfer.get());
+      if (multiapp_transfer)
+        if (multiapp_transfer->getMultiApp()->name().length() > max_multiapp_name_length)
+          max_multiapp_name_length = multiapp_transfer->getMultiApp()->name().length();
+    }
+    _console << std::left << std::setw(max_name_length + 1) << "Name";
+    _console << std::left << std::setw(max_type_length + 1) << "Type";
+    _console << std::left << std::setw(max_multiapp_name_length) << "MultiApp name\n";
+    unsigned int total_length = max_name_length + max_type_length + max_multiapp_name_length + 2;
+    for (unsigned int i = 0; i < total_length; ++i)
+      _console << '-';
+    _console << '\n';
+
+    for (const auto & transfer : transfers)
+    {
+      _console << std::left << std::setw(max_name_length + 1) << transfer->name();
+      _console << std::left << std::setw(max_type_length + 1) << transfer->type();
+      auto multiapp_transfer = dynamic_cast<MultiAppTransfer *>(transfer.get());
+      if (multiapp_transfer)
+        _console << std::left << std::setw(max_multiapp_name_length)
+                 << multiapp_transfer->getMultiApp()->name();
+      _console << "\n";
+    }
+    for (unsigned int i = 0; i < total_length; ++i)
+      _console << '-';
+    _console << std::endl;
```

Comment:
what does this end up looking like?

is there info about the execute_on ? That used to be provided since it was logging when executing

Reply:
Something like this
```
Starting Transfers on TIMESTEP_BEGIN From MultiApps
Name                    Type                                        MultiApp name
---------------------------------------------------------------------------------
pebble_receive_T_mod_0  MultiAppPostprocessorNearestElementTransfer pebble0      
pebble_receive_T_fuel_0 MultiAppPostprocessorNearestElementTransfer pebble0      
pebble_receive_T_mod_1  MultiAppPostprocessorNearestElementTransfer pebble1      
pebble_receive_T_fuel_1 MultiAppPostprocessorNearestElementTransfer pebble1      
pebble_receive_T_mod_2  MultiAppPostprocessorNearestElementTransfer pebble2      
pebble_receive_T_fuel_2 MultiAppPostprocessorNearestElementTransfer pebble2      
pebble_receive_T_mod_3  MultiAppPostprocessorNearestElementTransfer pebble3      
pebble_receive_T_fuel_3 MultiAppPostprocessorNearestElementTransfer pebble3      
pebble_receive_T_mod_4  MultiAppPostprocessorNearestElementTransfer pebble4      
pebble_receive_T_fuel_4 MultiAppPostprocessorNearestElementTransfer pebble4      
pebble_receive_T_mod_5  MultiAppPostprocessorNearestElementTransfer pebble5      
pebble_receive_T_fuel_5 MultiAppPostprocessorNearestElementTransfer pebble5      
pebble_receive_T_mod_6  MultiAppPostprocessorNearestElementTransfer pebble6      
pebble_receive_T_fuel_6 MultiAppPostprocessorNearestElementTransfer pebble6      
pebble_receive_T_mod_7  MultiAppPostprocessorNearestElementTransfer pebble7      
pebble_receive_T_fuel_7 MultiAppPostprocessorNearestElementTransfer pebble7      
pebble_receive_T_mod_8  MultiAppPostprocessorNearestElementTransfer pebble8      
pebble_receive_T_fuel_8 MultiAppPostprocessorNearestElementTransfer pebble8      
---------------------------------------------------------------------------------
Transfers on TIMESTEP_BEGIN Are Finished
```
with the first and last line colored.

Reply:
I was initially actually adding `direction` but then realized the header line contains the direction and execute_on information, so no need to add them in the table.